src/guides/v2.3/graphql/queries/category.md

```diff
@@ -212,3 +212,10 @@ The query returns a `CategoryTree` object, which implements [`CategoryInterface`
 Attribute | Data type | Description
 --- | --- | ---
 `children` | `CategoryTree` | An array containing the next level of subcategories. By default, you can specify up to 10 levels of child categories
+
+## Errors
+
+Error | Description
+--- | ---
+`Category doesn't exist` | The specified category-id value does not exist.
+`Field \"category\" argument \"id\" requires type Int, found "XXX" ` | The given category-id is not of type `int`, which is the required type.
```

Comment:
The suggestion is what other docs have said in this case - so consistency + should solve linting
```suggestion
`Field "category" argument "id" requires type Int, found "XXX" ` | The specified `id` argument value has the wrong type.
```